.github/workflows/sycl_precommit_linux.yml

```diff
@@ -1,7 +1,7 @@
 name: SYCL MLIR Pre Commit on Linux
 
 on:
-  pull_request_target:
+  pull_request:
```

Comment:
I think we need to drop lines 16-18 as well.

Reply:
removed.